client/components/deposits-overview/index.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Card, CardHeader } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Icon, moreVertical } from '@wordpress/icons';
+
+/**
+ * Internal dependencies.
+ */
+import NextDepositDetails from './next-deposit';
+import DepositsHistory from './history';
+import DepositsOverviewActions from './actions';
+import { useAllDepositsOverviews } from 'wcpay/data';
```

Comment:
I'd suggest just hard-coding placeholder content in this component, and let future PRs flesh that out as components. 

By adding placeholder components (and naming them) you're constraining future PRs - it's like a specification.

Reply:
Removed in [1ec0e0a](https://github.com/Automattic/woocommerce-payments/pull/5759/commits/1ec0e0a881a1cece7e2e8505e4b75806715fdc91)